catalog/transformer/catalog-transformer-rtf/pom.xml

```diff
@@ -0,0 +1,192 @@
+                                    <file>src/main/resources/default-category-dateTime.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-dateTime</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-location.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-location</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-media.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-media</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-security.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-security</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-topic.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-topic</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-validation.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-validation</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-version.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-version</classifier>
+                                </artifact>
+                            </artifacts>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Import-Package>
+                            *
+                        </Import-Package>
+                        <Export-Package>
+                            ddf.catalog.transformer.output.rtf, ddf.catalog.transformer.output.rtf.model
```

Comment:
✏️ Put these on separate lines for readability

Reply:
Agreed